### DISCUSSION TITLE
Delete branch button isn't available after using close branch feature

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#142434] [Mobile] [answered]
GitHub Mobile beta for Android 

Steps: 
1. Open a GitHub pull request in GitHub
2. Use the menu at the top right to close the PR 
3. See branch is closed but no button

![Screenshot_20241023-163136.png](https://github.com/user-attachments/assets/9ff64f11-1879-4022-8a5a-49326738d2ee)

Open PR in web browser and see button:
![Screenshot_20241023-163149.png](https://github.com/user-attachments/assets/94e20b14-4640-4b41-bde5-79dc86c0545f)

Reopen PR in app and see button:
![Screenshot_20241023-163348.png](https://github.com/user-attachments/assets/793e316b-29cd-4b97-8726-583f75ad205c)

**Comment by taki-on** [comment, marked as answer]
@jsoref Thank you for the report! I will file it as a bug and we will investigate. Thanks!
